R/transformCounts.R

```diff
@@ -45,10 +57,18 @@
 #'
 #' \itemize{
 #' 
-#' \item 'alr', 'chi.square', 'clr', 'frequency', 'hellinger', 'log', 'normalize', 'pa', 'rank', 'rclr'
+#' \item 'alr', 'chi.square', 'clr', 'frequency', 'hellinger', 'log', 
+#' 'normalize', 'pa', 'rank', 'rclr'
 #' 'relabundance', 'rrank', 'standardize', 'total': please refer to 
 #' \code{\link[vegan:decostand]{decostand}} for details.
 #' 
+#' \item 'css': Cumulative Sum Scaling (CSS) can be used to normalize count data 
+#' by accounting for differences in library sizes. By default, the function
+#' determines the normalization percentile for summing and scaling 
+#' counts. If you want to specify the percentile value, good default value
+#' might be \code{0.5}.The method is inspired by
+#' \code{\link[https://www.bioconductor.org/packages/metagenomeSeq/]{metagenomeSeq}}.
+#' 
```

Comment:
I removed "fast css" method as the only difference to this standard method is that the value of percentile is estimated from the data. Also "fast" did not describe the algorithm.

Now percentile is estimated from the data by default, but user can choose manually the percentile also.

Moreover, our code is different from metagenomeSeq (it does essentially the same thing, though). That is why the method is only "inspired" by metagenomeSeq

Reply:
The hard-coded test fail now and the results are different. Should we check or go ahead with the new changes?

Reply:
it's all good. Updated the percentile to get the right matrix. everything works now.